images/krte/cloudbuild.yaml

```diff
@@ -9,7 +9,6 @@ steps:
     - --build-arg=BAZEL_VERSION_ARG=$_BAZEL_VERSION
     - --build-arg=OLD_BAZEL_VERSION=$_OLD_BAZEL_VERSION
     - --build-arg=CFSSL_VERSION=$_CFSSL_VERSION
-    - --build-arg=UPGRADE_DOCKER_ARG=$_UPGRADE_DOCKER
```

Comment:
KRTE never used this, this is because we conflate the variants.yaml for the two images.
Something we shouldn't need to do long term as bazel is removed and go version can be installed based on .go-version